BeerOverflowWindowsApp/MainWindow.cs

```diff
@@ -204,10 +210,11 @@ private string GetRadius()
         private void ManualBarRating_Click(object sender, EventArgs e)
         {
             var rating = manualBarRating.Rating;
-            if (_selectedBar != null && rating != "" && int.TryParse(rating, out int ratingNumber))
+            int ratingNumber;
+            if (_selectedBar != null && rating != "" && int.TryParse(rating, out ratingNumber))
             {
-                Task.Run(() => _barRating.AddRating(_selectedBar, ratingNumber)).Wait();
-                Task.Run(() => _selectedBar.Ratings = new DatabaseManager().GetBarRatings(_selectedBar)).Wait();
```

Comment:
Is it possible to re-add the  "Task.Run(() => ...).Wait()" part for more responsivness?